tests/check-legal-aid/test/public-integration.json

```diff
@@ -0,0 +1,41 @@
+{
+  "src_folders" : "tests/nightwatch/specs",
+  "output_folder" : "tests/reports/nightwatch",
+  "custom_assertions_path" : "tests/nightwatch/custom_assertions",
+  "custom_commands_path" : "tests/nightwatch/commands",
+
+  "selenium" : {
+    "start_process" : true,
+    "log_path": "tests/reports",
+    "server_path" : "node_modules/selenium-standalone/.selenium/2.43.1/server.jar"
+  },
+
+  "test_settings" : {
+    "default" : {
+      "launch_url" : "http://public-integration.cla.dsd.io",
```

Comment:
Could we remove this configuration until it's necessary, please?

Reply:
:thumbsup: